libs/block-editor/src/lib/extensions/components/suggestions-list-item/suggestions-list-item.component.ts

```diff
@@ -23,7 +23,7 @@ export class SuggestionsListItemComponent implements FocusableOption, OnInit {
     constructor(private element: ElementRef) { }
 
     ngOnInit() {
-        this.icon = !(this.url.split('/').length > 1);
+        this.icon = this.icon = typeof( this.url ) === 'string' && !(this.url.split('/').length > 1);
```

Comment:
Whats is this for?

Reply:
Since the images are now base 64, we sanitize them using `domSanitizer` which returns a type of` SafeUrl`, which is not a `string`. This is why I make sure url is a string type before using the `split` method.